modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -29,7 +29,7 @@
 #' @description State Variable Data Assimilation: Ensemble Kalman Filter and Generalized ensemble filter. Check out SDA_control function for more details on the control arguments.
 #' 
 #' @return NONE
-#' @import nimble furrr
+#' @import nimble furrr purrr
```

Comment:
Similarly, don't import whole libraries in places where you can use namespaces instead